internal/release/postinstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
+# Copyright Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if command -v systemctl >/dev/null 2>&1; then
+    systemctl enable dynatrace-otel-collector.service
+    if [ -f /etc/dynatrace-otel-collector/config.yaml ]; then
+        systemctl start dynatrace-otel-collector.service
+    else
+      echo "Collector installed, but no config.yaml was found, skipping startup..."
```

Comment:
Are there docs somewhere where to put the config? Or do we have plans to add them somewhere? It might be confusing if the collector doesn't start because there is no config, but then there is no docs telling you where to put it.